paddlecloud/paddlejob/paddle_job.py

```diff
@@ -159,6 +159,8 @@ def new_trainer_job(self):
         }
         if self._registry_secret:
             job["spec"]["template"]["spec"].update({"imagePullSecrets": [{"name": self._registry_secret}]})
+        if self._gpu > 0:
+            job["spec"]["template"]["spec"]["containers"][0]["resources"]["limites"]["alpha.kubernetes.io/nvidia-gpu"] = str(self._gpu)
```

Comment:
We not only need to set the limit, the job Docker image also need to a GPU vesion.